lib/workers/repository/update/pr/index.ts

```diff
@@ -442,6 +442,8 @@ export async function ensurePr(
       } else if (!config.committedFiles && !config.rebaseRequested) {
         logger.debug(
           {
+            existingPrBodyHash,
+            newPrBodyHash,
```

Comment:
Is this actually helpful to the user? We don't intend for these hashes to be meaningful to anyone

Reply:
No, I added it when was debugging, removed